portal/src/components/FilterCategories/FilterCategories.component.js

```diff
@@ -115,34 +93,88 @@ export default {
         }
       )
     },
-    isShowCategoryItem({ category, item, indexItem }) {
-      return (
-        !item.is_empty && (category.showAll || indexItem < this.visible_items)
-      )
-    },
     getTitleTranslation(category, language) {
-      if (
-        !_.isNil(category.title_translations) &&
-        !_.isEmpty(category.title_translations)
-      ) {
+      if (category && category.title_translations) {
         return category.title_translations[language]
       }
       return category.name
+    },
+    datesRangeFilter() {
+      if (this.selectedFilters) {
+        const datesFilter = this.selectedFilters.find(
+          item => item.external_id === this.publisherDateExternalId
+        )
+        if (datesFilter && datesFilter.items) {
+          return {
+            start_date: datesFilter.items[0] || null,
+            end_date: datesFilter.items[1] || null
+          }
+        }
+      } else {
+        return {
+          start_date: null,
+          end_date: null
+        }
+      }
+    },
+    hasSelectedChildren(cat) {
+      return cat.children.filter(child => child.selected === true).length > 0
+    },
+    hasDatesRangeFilter(cat, selectedFilters) {
+      if (cat.external_id !== this.publisherDateExternalId) return false
+
+      const datesFilter = selectedFilters.find(
+        item => item.external_id === this.publisherDateExternalId
+      )
+
+      if (datesFilter && datesFilter.items) {
+        const items = datesFilter.items.filter(item => item !== null)
+        return items.length > 0
+      } else return false
+    },
+    sortFilterItems(items) {
+      const nullCounts = items.filter(item => item.count === null)
```

Comment:
Ik vind nog steeds dat null een `0` moet zijn hier

Reply:
Ja, denk het wel. Ik wil het even meepakken in die 'bol.com' filters als het niet teveel werk is.

Reply:
👍